lib/policy_machine_storage_adapters/active_record.rb

```diff
@@ -807,7 +849,7 @@ def associations_filtered_by_operation(associations, operation)
     # Builds an array of PolicyElement objects within the scope of a given
     # array of associations
     def build_accessible_object_scope(associations, options = {})
-      permitting_oas = PolicyElement.where(id: associations.pluck(:object_attribute_id))
+      permitting_oas = PolicyElement.where(id: associations.select(:object_attribute_id))
```

Comment:
it actually prevents an extra query, only possible because this is vanilla activerecord and those guys are really smart: 
```ruby
> PolicyElement.where(id: associations.select(:object_attribute_id)).to_sql
# => "SELECT \"policy_elements\".* FROM \"policy_elements\" WHERE \"policy_elements\".\"id\" IN (SELECT \"policy_element_associations\".\"object_attribute_id\" FROM \"policy_element_associations\" WHERE \"policy_element_associations\".\"user_attribute_id\" IN (2, 4) AND \"policy_element_associations\".\"operation_set_id\" = 17)"
> PolicyElement.where(id: associations.pluck(:object_attribute_id)).to_sql
# => "SELECT \"policy_elements\".* FROM \"policy_elements\" WHERE \"policy_elements\".\"id\" = 11"
```
note how `pluck` retrieves the id explicitly, meaning it loads it when it's resolving the entire line, while `select` generates into sql